docs/resources/elb_listener.md

```diff
@@ -60,6 +58,17 @@ The following arguments are supported:
   backend servers. The default value is false. This parameter is valid only when the protocol is set to *HTTP* or
   *HTTPS*.
 
+* `forward_port` - (Optional, Bool) Specifies whether transfer the listening port of the load balancer in the
+  X-Forwarded-Port header to backend servers. The default value is false. This parameter is valid only when the
+  protocol is set to *HTTP* or *HTTPS*.
+
+* `forward_request_port` - (Optional, Bool) Specifies whether transfer the source port of the client in the
+  X-Forwarded-For-Port header to backend servers. The default value is false. This parameter is valid only when the
+  protocol is set to *HTTP* or *HTTPS*.
+
+* `forward_host` - (Optional, Bool) Specifies whether rewrite the X-Forwarded-Host header. The default value is true.
+  This parameter is valid only when the protocol is set to *HTTP* or *HTTPS*.
```

Comment:
The description of `forward_host` is not clear, please add more details.

Reply:
done